src/main/java/cubicchunks/lighting/LightingManager.java

```diff
@@ -118,9 +118,19 @@ private boolean updateDiffuseLight(Cube cube, int localX, int localZ, int minY,
         }
         int blockX = localToBlock(cube.getX(), localX);
         int blockZ = localToBlock(cube.getZ(), localZ);
+        
+        CubePos minLoad = cube.getCoords().add(-1, -1, -1);
+        CubePos maxLoad = cube.getCoords().add(1, 1, 1);
```

Comment:
This is NOT equivalent to the code I had. Light updates need blocks in radius 17 blocks to be loaded (and there is currently a big in LightPropagator that makes the value even higher in some bad cases when reducing light value)